pkg/kubelet/disk_manager_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/cadvisor"
 )
 
+const mb = 1024 * 1024
```

Comment:
I like this.  Since this is redefined in disk_manager.go as well, how do you feel about making it a top level const in disk_manager and using that instance?